src/stan/mcmc/chains.hpp

```diff
@@ -701,6 +785,18 @@ namespace stan {
         return effective_sample_size(index(name));
       }
 
+      double effective_sample_size_pod(const int index) const {
+        std::vector<std::vector<double> > samples(num_chains());
+        for (int chain = 0; chain < num_chains(); chain++) {
+          Eigen::VectorXd Esample = this->samples(chain, index);
```

Comment:
Local variables shouldn't have upper-case names.  We reserve that for types in our own code (not that we've followed this everywhere.